org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
 {
-    "id": "org.musicbrainz.Picard",
+    "app-id": "org.musicbrainz.Picard",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
     "sdk": "org.freedesktop.Sdk",
     "command": "picard",
     "rename-desktop-file": "picard.desktop",
     "rename-icon": "picard",
     "finish-args": [
+        /* X11 + XShm access */
```

Comment:
Why add these comments? Those options are well-known, used by almost all apps, there is nothing special to them that requires documenting… (plus, comments are theoretically invalid in JSON, so I'd rather we avoid them as much as possible)